### ISSUE TITLE
Fix keyboard acessibility

### ISSUE THREAD

**Issue by tiago-neto1027** [tiago-neto1027/PRIMA-Chat-Application#46]
Several parts of the application are still not prepared to be keyboard only due to wrong TabIndexes or lack of Enter Keyboard shortcuts
